.grunt-config/webpack.js

```diff
@@ -136,6 +136,7 @@ const entry = {
 	'styleguide-app-initiator': path.resolve( __dirname, '../modules/styleguide/assets/js/styleguide-app-initiator.js' ),
 	'e-home-screen': path.resolve( __dirname, '../modules/home/assets/js/app.js' ),
 	'wc-product-editor': path.resolve( __dirname, '../assets/dev/js/admin/wc-product-editor.js' ),
+	'new-floating-elements': path.resolve( __dirname, '../assets/dev/js/admin/floating-elements/new-floating-elements.js' ),
```

Comment:
Do we need `new`?

Reply:
@louiswol94 I kept the naming convention of new-template, but I can remove it of course